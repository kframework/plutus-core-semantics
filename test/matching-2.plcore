(program

(module Foo (imported) (exported () ()) (local () ())
  (define result
    (case (con Nat.S (con Nat.Z))
          (Nat.S (x) (con Nat.Z))
          (Nat.Z ()  -1))))

)
