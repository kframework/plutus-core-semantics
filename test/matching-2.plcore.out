<T> <k> . </k> <lenv> .Map </lenv> <genv> ( Foo . result ) |-> ( con Nat . Z .ValList ) </genv> <mod> Foo </mod> <sym> .Map </sym> <gas> 9996 </gas> </T>
